src/views/product-tutorials-view/components/product-view-content/components/tutorials-stack/types.ts

```diff
@@ -2,12 +2,12 @@ import { Tutorial as ClientTutorial } from 'lib/learn-client/types'
 
 export interface TutorialsStackProps {
   /** Heading to show above the tutorial cards. */
-  heading: string
+  heading?: string
   /**
    * Identifier for the heading, which should unique in the context of the page
    * Note: headingSlug is added after fetching content from the Learn API
    */
-  headingSlug: string
+  headingSlug?: string
```

Comment:
[`heading` is not required for `TutorialsStack` blocks](https://github.com/hashicorp/dev-portal/blob/8e1f3770c6e361a38b27f4556e0683b3d6ada801/src/lib/learn-client/schemas.ts#L49), so these should in fact be optional props.